src/engines/nwn2/journal.h

```diff
@@ -0,0 +1,72 @@
+/* xoreos - A reimplementation of BioWare's Aurora engine
+ *
+ * xoreos is the legal property of its developers, whose names
+ * can be found in the AUTHORS file distributed with this source
+ * distribution.
+ *
+ * xoreos is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 3
+ * of the License, or (at your option) any later version.
+ *
+ * xoreos is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with xoreos. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/** @file
+ *  A journal in a Neverwinter Nights 2 module or campaign.
+ */
+
+#ifndef ENGINES_NWN2_JOURNAL_H
+#define ENGINES_NWN2_JOURNAL_H
+
+#include <vector>
+
+#include "src/common/ustring.h"
+
+namespace Engines {
+
+namespace NWN2 {
+
+class Journal {
+public:
+	Journal(const Aurora::GFF3Struct &journal);
+	~Journal();
```

Comment:
Since the destructor doesn't do anything special, you can mark that as

```cpp
~Journal() = default;
```

(and then remove the `Journal::~Journal()` from the .cpp file)